portal/src/model/themeAuthFlowV2.ts

```diff
@@ -298,7 +298,21 @@ abstract class StyleProperty<T> extends AbstractStyle<T> {
     this.value = value;
   }
 
-  abstract getCSSValue(): string | undefined;
+  getCSSValue(): string | undefined {
+    const raw = this.getRawCSSValue();
+    if (raw == null) {
+      return undefined;
+    }
+    // Looks like there are some problem in postcss:
+    // When you have a Declaration({prop: "someprop", value: "1px\\"})
+    // You add it to a `Root` object.
+    // And call `root.toString()`.
+    // The output will be { someprop: 1px\; } which missed one `\`
+    // While Declaration({prop: "someprop", value: "1px\\"}).toString() gives you `someprop: 1px\\;` which is correct.
+    // Actually there is no need to allow \ character in values, so we just remove them here.
+    return raw.replace(/\\/g, "");
+  }
+  abstract getRawCSSValue(): string | undefined;
```

Comment:
Per our offline discussion,

1. We will allow the input field to take any value.
2. However, we will validate the input field value. If the value is considered valid, then the CSS is updated immediately. Otherwise, the input field should be highlighted in red, the save button is disabled. When the value is invalid, in addition to the highlight state, there should be an error message saying why the value is invalid.

Reply:
https://github.com/user-attachments/assets/1ea3945b-116d-4cda-8441-d79346bef7b3


@louischan-oursky 
Updated, thanks!
- Added immediate validation logic on the form, which tells the user if the border radius is invalid.
- I did not validate color input. I made it accepting valid inputs instead.
- Removed code which remove the escape character, because now it should not exist in form state.